fhttp/http_client.go

```diff
@@ -732,6 +738,17 @@ func (c *FastClient) connect() net.Conn {
 	c.socketCount++
 	var socket net.Conn
 	var err error
+
+	// Resolve the DNS name when making new connections.
+	if !c.noResolveEachConn {
+		c.dest, err = resolve(c.hostname, c.port, c.resolve)
```

Comment:
doesn't that mean it's going to do it one extra time even for 1 connection? (once in client setup and once again at connect)

Reply:
good point, updated